### PR TITLE
Add initial tree branch rendering

### PR DESCRIPTION
**Pull request by Walt280**

### CODE REVIEW

File: src/branches/Leaf.ts [hunk 1]
```diff
@@ -54,9 +55,20 @@ export default class Leaf extends Container {
 		this.addChild(this.amountText);
 	}
 
-	private getTint(amount: number): [number, number | undefined] {
-		// TODO: Check date and return correct, color
-		return [0x8eb332, undefined];
+	private getTint(amount: number): [number, number | undefined, number] {
+		// Graduation sakura colors
+		if(Date.now() > new Date(2025, 0, 3).getTime()) {
```
Comment (GoldElysium) [inline src/branches/Leaf.ts]:
```suggestion
		if(Date.now() > new Date(2025, 0, 3).getTime() && Date.now() < new Date(2025, 0, 4).getTime()) {
```

File: src/branches/Leaf.ts [hunk 2]
```diff
@@ -54,9 +55,20 @@ export default class Leaf extends Container {
...
+				[0xFF8BD4, 0x000000, 1.75],
+				[0xFEB5E3, 0x000000, 1.75],
+				[0xFCBCDF, 0x000000, 1.75]
+			];
+
+			const randNum = getRandomNumber(0, sakuraColors.length);
+			return sakuraColors[randNum];
+		}
+
+		return [0x8eb332, undefined, 1];
```
Comment (GoldElysium) [inline src/branches/Leaf.ts]: Natural green still needs to be implemented

**Review by GoldElysium — CHANGES_REQUESTED** [review]

File: src/branches/Leaf.ts (hunk 2 above)
Comment (Walt280) [reply src/branches/Leaf.ts]: Implemented

File: src/branches/Leaf.ts (hunk 1 above)
Comment (Walt280) [reply src/branches/Leaf.ts]: Implemented

File: src/branches/Leaf.ts [hunk 3]
```diff
@@ -60,9 +78,32 @@ export default class Leaf extends Container {
 		this.addChild(this.amountText);
 	}
 
-	private getTint(amount: number): [number, number | undefined] {
-		// TODO: Check date and return correct, color
-		return [0x8eb332, undefined];
+	private getTint(amount: number): [number, number] {
+		// Graduation sakura colors
+		if(Date.now() > new Date(2025, 0, 3).getTime() && Date.now() < new Date(2025, 0, 4).getTime()) {
```
Comment (GoldElysium) [inline src/branches/Leaf.ts]:
```suggestion
		const now = new Date();
		if(now.getMonth() === 0 && now.getDate() === 3) {
```

**Review by GoldElysium — APPROVED** [review]

**Pull request merged**